pytidb/embeddings/builtin.py

```diff
@@ -203,7 +221,11 @@ def _process_image_source(self, source: SourceInputType) -> Union[str, dict]:
             else:
                 raise ValueError(f"invalid url format for image source: {source}")
         elif isinstance(source, Image):
-            return {"image": encode_pil_image_to_base64(source)}
+            base64_str = encode_pil_image_to_base64(source, compress_for_bedrock=self.model_name.startswith("bedrock/"))
```

Comment:
```suggestion
            base64_str = encode_pil_image_to_base64(source, max_base64_length)
```

I think the compress process is also applicable to other providers. The difference may be that the `max_base64_length` of image is different. Consider using the `max_base64_length` parameter instead of the `compress_for_bedrock` parameter, making the `encode_pil_image_to_base64` / `encode_file_to_base64` functions more general

Reply:
Changed as using `max_base64_length` to control the picture size.